apps/order/schema.py

```diff
@@ -0,0 +1,72 @@
+import graphene
+from graphene_django import DjangoObjectType
+from graphene_django_extras import PageGraphqlPagination
+from django.db.models import QuerySet
+from django.db.models import F, Sum
+
+from utils.graphene.types import CustomDjangoListObjectType
+from utils.graphene.fields import DjangoPaginatedListObjectField
+
+from apps.order.models import CartItem
+
+
+def get_cart_items_qs(info):
+    return CartItem.objects.filter(created_by=info.context.user).annotate(
+        total_price=F('book__price') * F('quantity')
+    )
+
+
+class CartItemType(DjangoObjectType):
+    total_price = graphene.Int()
+
+    class Meta:
+        model = CartItem
+        fields = ('id', 'book', 'quantity', 'total_price')
+
+    @staticmethod
+    def get_custom_queryset(queryset, info, **kwargs):
+        return get_cart_items_qs(info)
+
+    @staticmethod
+    def resolve_total_price(root, info, **kwargs) -> QuerySet:
+        return root.total_price
+
+
+class CartGrandTotalType(graphene.ObjectType):
```

Comment:
Can't we include this in single schema that we have for cart items?

Reply:
@k9845  We created CartGrandTotalType to take grand total outside paginated response results.